src/tracing/tracing_args.gni

```diff
@@ -25,6 +25,10 @@ declare_args() {
   # backends explicitly
   matter_enable_tracing_support = current_os == "android"
 
+  if (chip_device_platform == "darwin") {
```

Comment:
place this above as: `matter_enable_tracing_support = (current_os == "android") || (chip_device_platform == "darwin")`

Reply:
@anush-apple ?

Reply:
Adopted suggestion